idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/mode/iconpack/creation/common/inputhandler/InputHandler.kt

```diff
@@ -0,0 +1,116 @@
+package io.github.composegears.valkyrie.ui.screen.mode.iconpack.creation.common.inputhandler
+
+import io.github.composegears.valkyrie.ui.domain.validation.IconPackValidationUseCase
+import io.github.composegears.valkyrie.ui.domain.validation.InputState
+import io.github.composegears.valkyrie.ui.domain.validation.PackageValidationUseCase
+import io.github.composegears.valkyrie.ui.domain.validation.ValidationResult
+import io.github.composegears.valkyrie.ui.extension.updateState
+import io.github.composegears.valkyrie.ui.screen.mode.iconpack.creation.common.packedit.model.InputChange
+import io.github.composegears.valkyrie.ui.screen.mode.iconpack.creation.common.packedit.model.InputFieldState
+import io.github.composegears.valkyrie.ui.screen.mode.iconpack.creation.common.packedit.model.NestedPack
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+interface InputHandler {
```

Comment:
```suggestion
sealed interface InputHandler {
```

Reply:
looks useless

also need to be in single package
<img width="960" alt="image" src="https://github.com/user-attachments/assets/f74a99b8-ca18-433b-a3ee-ac17ebf53b6a">

Reply:
Alright, that's it.